client/signup/steps/rewind-confirm/index.jsx

```diff
@@ -0,0 +1,87 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import StepWrapper from 'signup/step-wrapper';
+import Card from 'components/card';
+import Button from 'components/button';
+import SignupActions from 'lib/signup/actions';
+import HappychatButton from 'components/happychat/button';
+
+class RewindConfirmStep extends Component {
+	static propTypes = {
+		flowName: PropTypes.string,
+		goToNextStep: PropTypes.func.isRequired,
+		positionInFlow: PropTypes.number,
+		signupProgress: PropTypes.array,
+		stepName: PropTypes.string,
+	};
+
+	addCredentials = () => {
+		SignupActions.submitSignupStep( {
+			processingMessage: this.props.translate( 'Setting up your site' ),
+			stepName: this.props.stepName,
+		} );
+
+		this.props.goToNextStep();
+	};
+
+	skipStep = () => {
+		SignupActions.submitSignupStep( {
+			processingMessage: this.props.translate( 'Setting up your site' ),
+			stepName: this.props.stepName,
+		} );
+
+		this.props.goToStep( 'rewind-complete' );
+	};
+
+	contactSupport = () => alert( 'Not yet implemented' );
+
+	renderStepContent = () => {
+		const { translate } = this.props;
+
+		return (
+			<Card className="rewind-confirm__card">
+				<h3 className="rewind-confirm__title">{ translate( 'Are you sure?' ) }</h3>
+				<img className="rewind-confirm__image" src="/calypso/images/illustrations/security.svg" />
+				<p className="rewind-confirm__description">
+					{ translate(
+						"If you don't share credentials with Jetpack, your site won't be backed up. Our " +
+							'support staff is available to answer any questions you might have.'
+					) }
+				</p>
+				<HappychatButton className="rewind-confirm__happychat-button" borderless={ false }>
+					{ translate( 'Contact support' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **35** times:
`translate( 'Contact Support' )` *ES Score: 15*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/rewind/signup-flow-get-creds)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).